exchange/exchange_test.go

```diff
@@ -2449,31 +2449,35 @@ func newExchangeForTests(t *testing.T, filename string, expectations map[string]
 	}
 }
 
-type mockBidIDGenerator struct {
+type fakeBidIDGenerator struct {
 	GenerateBidID bool `json:"generateBidID"`
 	ReturnError   bool `json:"returnError"`
+	bidCount      map[string]int
 }
 
-func (big *mockBidIDGenerator) Enabled() bool {
-	return big.GenerateBidID
+func (m *fakeBidIDGenerator) Enabled() bool {
```

Comment:
Should this variable name be `f`? I would assume `m` would be if the generator was still called `mock`.

Reply:
Sure. Change made.